dev-util/phpstorm/phpstorm-2017.3.4.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils vcs-snapshot
+
+DESCRIPTION="IDE for PHP built on the JetBrains IntelliJ IDEA platform"
+HOMEPAGE="https://www.jetbrains.com/phpstorm/"
+SRC_URI="https://download-cf.jetbrains.com/idea/PhpStorm-${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="|| ( IDEA IDEA_Academic IDEA_Classroom IDEA_OpenSource IDEA_Personal )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="virtual/jre"
```

Comment:
For some time now, PhpStorm requires Java 1.8+. It would be best to make that clear here in `RDEPEND`.